R/labelled.R

```diff
@@ -88,7 +88,7 @@ make_labelled <- function(supertbl, format_labels = NULL) {
 
   ## Set some predefined labels for data fields that aren't in the metadata
   data_labs <- c( # nolint: object_usage_linter
-    redcap_repeat_instance = "REDCap Repeat Instance",
+    redcap_form_instance = "REDCap Repeat Instance",
```

Comment:
To note this will need to be updated to reflect "REDCap Form Instance" and expanded to also include "REDCap Event Instance" if it exists.